scss/product/components/_modal.scss

```diff
@@ -81,7 +81,9 @@
 }
 // Actual modal
 .modal-content {
-  max-height: 100vh;
+  // for avoiding overflow of modal
+  // as we are calling 8% margin for modal container we cant have height more than 84vh
+  max-height: 85vh; 
```

Comment:
8% margin, then max-height should be `max-height: 84vh;` Right @rakeshyc

Reply:
will update @is-ashish